include/rocksdb/listener.h

```diff
@@ -143,6 +143,15 @@ struct TableFileDeletionInfo {
   Status status;
 };
 
+struct FileOperationInfo {
+  std::string path;
```

Comment:
Can it be `const std::string&` to avoid a string copying?

Reply:
I think so, will update. Thanks!